runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -1715,6 +1731,14 @@ private void updateThreadMetrics() {
     activeThreads.addValue(workUnitExecutor.activeCount());
     totalAllocatedThreads.getAndReset();
     totalAllocatedThreads.addValue(chooseMaximumNumberOfThreads());
+    outstandingBytes.getAndReset();
+    outstandingBytes.addValue(workUnitExecutor.bytesOutstanding());
+    maxOutstandingBytes.getAndReset();
+    maxOutstandingBytes.addValue(workUnitExecutor.maximumBytesOutstanding());
+    outstandingBundles.getAndReset();
+    outstandingBundles.addValue(workUnitExecutor.elementsOutstanding());
```

Comment:
a little unclear here that bundles are also referred to as elements. Is this supposed to be the case?

Reply:
The elements was the variable defined already in the BoundedQueueExecutor, should I change this naming?

Reply:
I will just leave the naming based on how the BoundedQueueExecutor has named everything.